gulpfile.js

```diff
@@ -335,24 +335,35 @@ function checkChromePreferencesFile(chromePrefsPath, webPrefs) {
   const webPrefsKeys = Object.keys(webPrefs);
   webPrefsKeys.sort();
```

Comment:
This looks unused now, since you're no longer accessing `webPrefsKeys` below.

Reply:
Also, given the other changes in this patch it seems that it's probably not necessary to sort the `chromePrefsKeys` any more and the following line can be removed as well?
https://github.com/mozilla/pdf.js/blob/50488d7a47299f3843c77df1d180265bc3817bc2/gulpfile.js#L330